Unable to apply theme in WSL Ubuntu

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What happened?

I followed the installation instruction on the website:

`curl -s https://ohmyposh.dev/install.sh | bash -s`

However, I was prompted to used `sudo` to run the bash command. So I have to change it to 

`curl -s https://ohmyposh.dev/install.sh | sudo bash -s`

and the installation is successful 

![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/31999415/8da911eb-4477-4c4f-9611-558dff8113e2)

However, as suggested by the instruction, when I put the customization config into my `.bashrc` file, I got into "config" error and the theme is not loaded correctly.

`eval "$(oh-my-posh init bash --config /root/themes/peru.omp.json)"`

![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/31999415/89cdb6ed-d4a1-49c2-b711-ba460f7f6af4)

I checked the folder `/root/themes` and the themes are indeed there:

![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/31999415/cac2593b-2ab7-4406-ad40-1007cc4a0dbe)

Finally, I was able to apply themes using remote config. But no luck with local theme files

`eval "$(oh-my-posh init bash --config 'https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh/main/themes/peru.omp.json')"`


### Theme

peru.omp.json

### What OS are you seeing the problem on?

Linux

### Which shell are you using?

bash

### Log output

```Shell
Version: 18.26.1

Shell: bash (5.0.17(1)-release)

Prompt:

 xushen   ~  CONFIG ERROR                                                                                in bash at 21:36:22
Segments:

ConsoleTitle(true)                        -   0 ms
Session(true)                             -   3 ms
Path(true)                                -   4 ms
Git(false)                                -   7 ms
Root(false)                               -   0 ms
Exit(true)                                -   2 ms
Node(false)                               -   0 ms
Go(false)                                 -   0 ms
Python(false)                             -   0 ms
Shell(true)                               -   0 ms
Time(true)                                -   0 ms

Run duration: 14.911032ms

Cache path: /home/xushen/.cache/oh-my-posh

Config path: /root/themes/peru.omp.json

Logs:

[DEBUG] 21:36:22.973 shell.go:Getenv:398 → NO DATA
[TRACE] 21:36:22.973 shell.go:Getenv(TERM_PROGRAM) - 204.414µs
[DEBUG] 21:36:22.973 shell.go:Getenv:398 → NO DATA
[TRACE] 21:36:22.973 shell.go:Getenv(OMP_CACHE_DIR) - 5.158µs
[DEBUG] 21:36:22.973 shell.go:Getenv:398 → NO DATA
[TRACE] 21:36:22.973 shell.go:Getenv(XDG_CACHE_HOME) - 3.22µs
[TRACE] 21:36:22.973 shell_unix.go:CachePath() - 30.033µs
[DEBUG] 21:36:22.974 shell.go:Getenv:398 → /root/themes/peru.omp.json
[TRACE] 21:36:22.974 shell.go:Getenv(POSH_THEME) - 6.803µs
[DEBUG] 21:36:22.974 shell_unix.go:Platform:77 → ubuntu
[TRACE] 21:36:22.974 shell.go:resolveConfigPath() - 23.53µs
[TRACE] 21:36:22.974 shell.go:Init() - 913.473µs
[TRACE] 21:36:22.974 shell.go:Flags() - 78ns
[ERROR] 21:36:22.974 config.go:loadConfig:160 → open /root/themes/peru.omp.json: permission denied
[TRACE] 21:36:22.974 config.go:loadConfig() - 65.303µs
[TRACE] 21:36:22.974 shell.go:Flags() - 79ns
[DEBUG] 21:36:22.974 shell.go:Getenv:398 → NO DATA
[TRACE] 21:36:22.974 shell.go:Getenv(OMP_CACHE_DISABLED) - 3.572µs
[DEBUG] 21:36:22.974 shell.go:Shell:652 → no shell name provided in flags, trying to detect it
[DEBUG] 21:36:22.977 shell.go:Shell:660 → process name: bash
[TRACE] 21:36:22.977 shell.go:Shell() - 2.797713ms
[DEBUG] 21:36:22.977 shell.go:Getenv:398 → 5.0.17(1)-release
[TRACE] 21:36:22.977 shell.go:Getenv(POSH_SHELL_VERSION) - 4.322µs
[DEBUG] 21:36:22.977 debug.go:PrintDebug:22 → Segment: Title
[DEBUG] 21:36:22.977 text.go:Render:70 → Rendering template: {{ .Shell }} in {{ .Folder }}
[TRACE] 21:36:22.977 shell_unix.go:Root() - 676ns
[TRACE] 21:36:22.977 shell.go:Shell() - 161ns
[TRACE] 21:36:22.977 shell.go:StatusCodes() - 64ns
[DEBUG] 21:36:22.977 shell_unix.go:IsWsl:35 → true
[TRACE] 21:36:22.977 shell_unix.go:IsWsl() - 9.371µs
[DEBUG] 21:36:22.977 shell.go:TemplateCache:833 → environment: [SHELL=/bin/bash WSL2_GUI_APPS_ENABLED=1 POSH_PID=11095 WSL_DISTRO_NAME=Ubuntu-20.04 WT_SESSION=65d857d4-1a84-4ba9-a1fb-2f881fd340ef NAME=XS-GRAM-23 PWD=/home/xushen LOGNAME=xushen HOME=/home/xushen LANG=C.UTF-8 WSL_INTEROP=/run/WSL/11089_interop LS_COLORS=rs=0:di=01;34:ln=01;36:mh=00:pi=40;33:so=01;35:do=01;35:bd=40;33;01:cd=40;33;01:or=40;31;01:mi=00:su=37;41:sg=30;43:ca=30;41:tw=30;42:ow=34;42:st=37;44:ex=01;32:*.tar=01;31:*.tgz=01;31:*.arc=01;31:*.arj=01;31:*.taz=01;31:*.lha=01;31:*.lz4=01;31:*.lzh=01;31:*.lzma=01;31:*.tlz=01;31:*.txz=01;31:*.tzo=01;31:*.t7z=01;31:*.zip=01;31:*.z=01;31:*.dz=01;31:*.gz=01;31:*.lrz=01;31:*.lz=01;31:*.lzo=01;31:*.xz=01;31:*.zst=01;31:*.tzst=01;31:*.bz2=01;31:*.bz=01;31:*.tbz=01;31:*.tbz2=01;31:*.tz=01;31:*.deb=01;31:*.rpm=01;31:*.jar=01;31:*.war=01;31:*.ear=01;31:*.sar=01;31:*.rar=01;31:*.alz=01;31:*.ace=01;31:*.zoo=01;31:*.cpio=01;31:*.7z=01;31:*.rz=01;31:*.cab=01;31:*.wim=01;31:*.swm=01;31:*.dwm=01;31:*.esd=01;31:*.jpg=01;35:*.jpeg=01;35:*.mjpg=01;35:*.mjpeg=01;35:*.gif=01;35:*.bmp=01;35:*.pbm=01;35:*.pgm=01;35:*.ppm=01;35:*.tga=01;35:*.xbm=01;35:*.xpm=01;35:*.tif=01;35:*.tiff=01;35:*.png=01;35:*.svg=01;35:*.svgz=01;35:*.mng=01;35:*.pcx=01;35:*.mov=01;35:*.mpg=01;35:*.mpeg=01;35:*.m2v=01;35:*.mkv=01;35:*.webm=01;35:*.ogm=01;35:*.mp4=01;35:*.m4v=01;35:*.mp4v=01;35:*.vob=01;35:*.qt=01;35:*.nuv=01;35:*.wmv=01;35:*.asf=01;35:*.rm=01;35:*.rmvb=01;35:*.flc=01;35:*.avi=01;35:*.fli=01;35:*.flv=01;35:*.gl=01;35:*.dl=01;35:*.xcf=01;35:*.xwd=01;35:*.yuv=01;35:*.cgm=01;35:*.emf=01;35:*.ogv=01;35:*.ogx=01;35:*.aac=00;36:*.au=00;36:*.flac=00;36:*.m4a=00;36:*.mid=00;36:*.midi=00;36:*.mka=00;36:*.mp3=00;36:*.mpc=00;36:*.ogg=00;36:*.ra=00;36:*.wav=00;36:*.oga=00;36:*.opus=00;36:*.spx=00;36:*.xspf=00;36: POSH_SHELL_VERSION=5.0.17(1)-release WAYLAND_DISPLAY=wayland-0 CONDA_PROMPT_MODIFIER=false LESSCLOSE=/usr/bin/lesspipe %s %s TERM=xterm-256color LESSOPEN=| /usr/bin/lesspipe %s USER=xushen DISPLAY=:0 SHLVL=1 POSH_THEME=/root/themes/peru.omp.json XDG_RUNTIME_DIR=/run/user/1000/ WSLENV=WT_SESSION:WT_PROFILE_ID: XDG_DATA_DIRS=/usr/local/share:/usr/share:/var/lib/snapd/desktop PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/usr/lib/wsl/lib:/mnt/c/Windows/system32:/mnt/c/Windows:/mnt/c/Windows/System32/Wbem:/mnt/c/Windows/System32/WindowsPowerShell/v1.0/:/mnt/c/Windows/System32/OpenSSH/:/mnt/c/Program Files/Git/cmd:/mnt/c/Users/xushe/AppData/Local/Microsoft/WindowsApps:/mnt/c/Users/xushe/AppData/Local/Programs/oh-my-posh/bin:/mnt/c/Users/xushe/AppData/Local/Programs/Microsoft VS Code/bin:/snap/bin DBUS_SESSION_BUS_ADDRESS=unix:path=/run/user/1000/bus HOSTTYPE=x86_64 POWERLINE_COMMAND=oh-my-posh PULSE_SERVER=unix:/mnt/wslg/PulseServer WT_PROFILE_ID={4dd1e689-b517-5f39-947d-78e8a8bdf958} _=/usr/local/bin/oh-my-posh]
[TRACE] 21:36:22.977 shell.go:GOOS() - 128ns
[DEBUG] 21:36:22.977 shell.go:Pwd:428 → /home/xushen
[TRACE] 21:36:22.977 shell.go:Pwd() - 116.757µs
[TRACE] 21:36:22.977 shell.go:GOOS() - 95ns
[TRACE] 21:36:22.977 shell.go:GOOS() - 62ns
[TRACE] 21:36:22.977 shell.go:GOOS() - 42ns
[TRACE] 21:36:22.977 shell.go:GOOS() - 39ns
[TRACE] 21:36:22.977 shell.go:GOOS() - 40ns
[TRACE] 21:36:22.977 shell.go:GOOS() - 41ns
[TRACE] 21:36:22.977 shell.go:GOOS() - 41ns
[TRACE] 21:36:22.977 shell.go:GOOS() - 40ns
[DEBUG] 21:36:22.977 shell.go:User:563 → xushen
[TRACE] 21:36:22.977 shell.go:User() - 3.22µs
[DEBUG] 21:36:22.977 shell.go:Host:575 → XS-GRAM-23
[TRACE] 21:36:22.977 shell.go:Host() - 4.654µs
[TRACE] 21:36:22.977 shell.go:GOOS() - 41ns
[DEBUG] 21:36:22.977 shell_unix.go:Platform:77 → ubuntu
[DEBUG] 21:36:22.977 shell.go:Getenv:398 → 1
[TRACE] 21:36:22.977 shell.go:Getenv(SHLVL) - 2.937µs
[TRACE] 21:36:22.977 shell.go:TemplateCache() - 244.168µs
[DEBUG] 21:36:22.977 shell.go:Getenv:398 → NO DATA
[TRACE] 21:36:22.977 shell.go:Getenv(POSH_CURSOR_LINE) - 4.401µs
[DEBUG] 21:36:22.977 shell.go:Getenv:398 → NO DATA
[TRACE] 21:36:22.977 shell.go:Getenv(POSH_CURSOR_COLUMN) - 2.543µs
[TRACE] 21:36:22.977 shell.go:Flags() - 59ns
[TRACE] 21:36:22.977 shell.go:Flags() - 63ns
[TRACE] 21:36:22.977 shell.go:Shell() - 106ns
[TRACE] 21:36:22.977 shell.go:Shell() - 37ns
[TRACE] 21:36:22.977 shell.go:Flags() - 103ns
[TRACE] 21:36:22.977 shell.go:Flags() - 700ns
[DEBUG] 21:36:22.977 properties.go:GetString:28 → {{ .Code }}
[TRACE] 21:36:22.977 shell.go:Pwd() - 190ns
[TRACE] 21:36:22.977 shell.go:GOOS() - 69ns
[DEBUG] 21:36:22.977 segment.go:SetEnabled:512 → Segment: Exit
[TRACE] 21:36:22.977 shell.go:Pwd() - 200ns
[TRACE] 21:36:22.977 shell.go:StatusCodes() - 159ns
[TRACE] 21:36:22.977 shell.go:GOOS() - 84ns
[DEBUG] 21:36:22.977 segment.go:SetEnabled:512 → Segment: Path
[TRACE] 21:36:22.978 shell.go:Pwd() - 90ns
[TRACE] 21:36:22.978 shell.go:Shell() - 156ns
[TRACE] 21:36:22.978 shell.go:Shell() - 68ns
[DEBUG] 21:36:22.978 properties.go:GetBool:22 → mapped_locations_enabled: true
[DEBUG] 21:36:22.978 properties.go:GetString:28 → 
[DEBUG] 21:36:22.978 properties.go:GetString:28 → 
[TRACE] 21:36:22.978 shell.go:GOOS() - 82ns
[DEBUG] 21:36:22.978 properties.go:GetString:28 → ~
[DEBUG] 21:36:22.978 properties.go:GetKeyValueMap:46 → mapped_locations: map[]
[TRACE] 21:36:22.978 shell.go:PathSeparator() - 224ns
[TRACE] 21:36:22.978 shell.go:GOOS() - 64ns
[TRACE] 21:36:22.978 shell.go:GOOS() - 64ns
[TRACE] 21:36:22.978 shell.go:Flags() - 455ns
[TRACE] 21:36:22.978 shell.go:GOOS() - 636ns
[TRACE] 21:36:22.978 shell.go:GOOS() - 60ns
[TRACE] 21:36:22.978 shell.go:PathSeparator() - 59ns
[TRACE] 21:36:22.978 shell.go:Flags() - 192ns
[TRACE] 21:36:22.978 shell.go:PathSeparator() - 73ns
[TRACE] 21:36:22.978 shell.go:GOOS() - 59ns
[TRACE] 21:36:22.978 shell.go:GOOS() - 59ns
[TRACE] 21:36:22.978 shell.go:PathSeparator() - 54ns
[TRACE] 21:36:22.978 shell.go:GOOS() - 65ns
[TRACE] 21:36:22.978 shell.go:GOOS() - 112ns
[TRACE] 21:36:22.978 shell.go:PathSeparator() - 57ns
[TRACE] 21:36:22.978 shell.go:GOOS() - 58ns
[TRACE] 21:36:22.978 shell.go:Pwd() - 258ns
[TRACE] 21:36:22.978 shell.go:GOOS() - 34ns
[TRACE] 21:36:22.978 shell.go:GOOS() - 121ns
[TRACE] 21:36:22.978 shell.go:PathSeparator() - 63ns
[TRACE] 21:36:22.978 shell.go:Pwd() - 129ns
[TRACE] 21:36:22.978 shell.go:Pwd() - 183ns
[DEBUG] 21:36:22.978 shell_unix.go:IsWsl:35 → true
[TRACE] 21:36:22.978 shell.go:GOOS() - 60ns
[TRACE] 21:36:22.978 shell_unix.go:IsWsl() - 6.808µs
[DEBUG] 21:36:22.978 segment.go:SetEnabled:512 → Segment: Git
[DEBUG] 21:36:22.978 shell.go:Getenv:398 → NO DATA
[TRACE] 21:36:22.978 shell.go:Getenv(SSH_CONNECTION) - 14.471µs
[DEBUG] 21:36:22.978 shell.go:Getenv:398 → NO DATA
[TRACE] 21:36:22.978 shell.go:Getenv(SSH_CLIENT) - 2.396µs
[DEBUG] 21:36:22.978 shell_unix.go:IsWsl:35 → true
[DEBUG] 21:36:22.978 shell_unix.go:Platform:77 → ubuntu
[TRACE] 21:36:22.978 shell_unix.go:IsWsl() - 6.86µs
[TRACE] 21:36:22.980 shell.go:Flags() - 206ns
[TRACE] 21:36:22.980 shell.go:Pwd() - 360ns
[TRACE] 21:36:22.980 shell.go:GOOS() - 137ns
[DEBUG] 21:36:22.980 segment.go:SetEnabled:512 → Segment: Root
[TRACE] 21:36:22.980 shell_unix.go:Root() - 868ns
[TRACE] 21:36:22.980 shell.go:TemplateCache() - 970ns
[DEBUG] 21:36:22.980 properties.go:GetBool:22 → always_enabled: true
[TRACE] 21:36:22.980 shell.go:TemplateCache() - 307ns
[DEBUG] 21:36:22.981 shell.go:RunCommand:593 → NO DATA
[TRACE] 21:36:22.981 shell.go:RunCommand(who am i) - 3.018409ms
[TRACE] 21:36:22.981 shell.go:TemplateCache() - 572ns
[DEBUG] 21:36:22.981 shell.go:RunCommand:593 → //wsl.localhost/Ubuntu-20.04/home/xushen
[TRACE] 21:36:22.981 shell.go:RunCommand(wslpath -m /home/xushen) - 3.26356ms
[TRACE] 21:36:22.981 shell_unix.go:IsWsl2() - 3.282403ms
[TRACE] 21:36:22.981 shell.go:Pwd() - 215ns
[TRACE] 21:36:22.981 shell.go:StackCount() - 279ns
[TRACE] 21:36:22.981 shell_unix.go:DirIsWritable(/home/xushen) - 9.647µs
[TRACE] 21:36:22.981 shell.go:TemplateCache() - 502ns
[DEBUG] 21:36:22.985 shell.go:RunCommand:593 → //wsl.localhost/Ubuntu-20.04/home/xushen
[TRACE] 21:36:22.985 shell.go:RunCommand(wslpath -m /home/xushen) - 3.56494ms
[TRACE] 21:36:22.985 shell.go:GOOS() - 44ns
[DEBUG] 21:36:22.985 shell.go:CommandPath:614 → /usr/bin/git
[TRACE] 21:36:22.985 shell.go:CommandPath(git) - 15.724µs
[TRACE] 21:36:22.985 shell.go:HasCommand(git) - 17.725µs
[TRACE] 21:36:22.985 shell.go:Pwd() - 67ns
[ERROR] 21:36:22.985 shell.go:HasParentFilePath:748 → stat .git: no such file or directory
[TRACE] 21:36:22.985 shell.go:HasParentFilePath(.git) - 72.709µs
[DEBUG] 21:36:22.985 properties.go:GetBool:22 → fetch_bare_info: false
[DEBUG] 21:36:22.985 text.go:Render:70 → Rendering template:  CONFIG ERROR
[TRACE] 21:36:22.985 shell.go:Shell() - 232ns
[DEBUG] 21:36:22.985 text.go:Render:70 → Rendering template:  {{ if .SSHSession }} {{ end }}{{ .UserName }}
[DEBUG] 21:36:22.985 text.go:Render:70 → Rendering template:   {{ path .Path .Location }}
[TRACE] 21:36:22.985 shell.go:TemplateCache() - 317ns
[TRACE] 21:36:22.985 shell.go:Shell() - 128ns
[TRACE] 21:36:22.986 shell.go:TemplateCache() - 205ns
[TRACE] 21:36:22.986 shell.go:Shell() - 60ns
[DEBUG] 21:36:22.986 text.go:Render:70 → Rendering template: diamond
[DEBUG] 21:36:22.986 text.go:Render:70 → Rendering template: powerline
[DEBUG] 21:36:22.986 text.go:Render:70 → Rendering template: powerline
[DEBUG] 21:36:22.986 text.go:Render:70 → Rendering template: powerline
[DEBUG] 21:36:22.986 text.go:Render:70 → Rendering template: p:red
[DEBUG] 21:36:22.986 text.go:Render:70 → Rendering template: diamond
[TRACE] 21:36:22.986 shell.go:Shell() - 91ns
[TRACE] 21:36:22.986 shell.go:Shell() - 24ns
[TRACE] 21:36:22.986 shell.go:Shell() - 23ns
[TRACE] 21:36:22.986 shell.go:Shell() - 24ns
[DEBUG] 21:36:22.986 shell.go:Getenv:398 → NO DATA
[TRACE] 21:36:22.986 shell.go:Getenv(OMP_CACHE_DISABLED) - 2.807µs
[TRACE] 21:36:22.986 shell.go:Flags() - 39ns
[TRACE] 21:36:22.986 shell.go:Flags() - 74ns
[TRACE] 21:36:22.986 shell.go:Pwd() - 80ns
[TRACE] 21:36:22.986 shell.go:GOOS() - 47ns
[DEBUG] 21:36:22.986 segment.go:SetEnabled:512 → Segment: Time
[DEBUG] 21:36:22.986 properties.go:GetString:28 → 15:04:05
[TRACE] 21:36:22.986 shell.go:TemplateCache() - 167ns
[TRACE] 21:36:22.986 shell.go:Flags() - 230ns
[TRACE] 21:36:22.986 shell.go:Flags() - 53ns
[DEBUG] 21:36:22.986 properties.go:GetString:28 → files
[TRACE] 21:36:22.986 shell.go:Pwd() - 44ns
[TRACE] 21:36:22.986 shell.go:GOOS() - 52ns
[TRACE] 21:36:22.986 shell.go:GOOS() - 31ns
[DEBUG] 21:36:22.986 segment.go:SetEnabled:512 → Segment: Python
[DEBUG] 21:36:22.986 segment.go:SetEnabled:512 → Segment: Go
[DEBUG] 21:36:22.986 properties.go:GetStringArray:52 → extensions: [*.go go.mod]
[TRACE] 21:36:22.986 shell.go:Flags() - 164ns
[DEBUG] 21:36:22.986 properties.go:GetStringArray:52 → folders: [.venv venv virtualenv venv-win pyenv-win]
[DEBUG] 21:36:22.986 properties.go:GetStringArray:52 → folders: []
[DEBUG] 21:36:22.986 properties.go:GetBool:22 → home_enabled: false
[TRACE] 21:36:22.986 shell.go:Pwd() - 74ns
[TRACE] 21:36:22.986 shell.go:Pwd() - 68ns
[TRACE] 21:36:22.986 shell.go:Pwd() - 81ns
[TRACE] 21:36:22.986 shell.go:Flags() - 51ns
[TRACE] 21:36:22.986 shell.go:GOOS() - 58ns
[TRACE] 21:36:22.986 shell.go:Pwd() - 73ns
[DEBUG] 21:36:22.986 segment.go:SetEnabled:512 → Segment: Shell
[TRACE] 21:36:22.986 shell.go:GOOS() - 71ns
[DEBUG] 21:36:22.986 segment.go:SetEnabled:512 → Segment: Node
[DEBUG] 21:36:22.986 properties.go:GetKeyValueMap:46 → mapped_shell_names: map[]
[TRACE] 21:36:22.986 shell.go:Shell() - 48ns
[TRACE] 21:36:22.986 shell.go:Flags() - 56ns
[DEBUG] 21:36:22.986 properties.go:GetStringArray:52 → extensions: [*.js *.ts package.json .nvmrc pnpm-workspace.yaml .pnpmfile.cjs .npmrc .vue]
[TRACE] 21:36:22.986 shell.go:TemplateCache() - 186ns
[DEBUG] 21:36:22.986 properties.go:GetStringArray:52 → folders: []
[DEBUG] 21:36:22.986 properties.go:GetBool:22 → home_enabled: false
[TRACE] 21:36:22.986 shell.go:Pwd() - 69ns
[DEBUG] 21:36:22.986 text.go:Render:70 → Rendering template: at <p:blue><b>{{ .CurrentDate | date "15:04:05" }}</b></>
[DEBUG] 21:36:22.986 text.go:Render:70 → Rendering template: in <p:blue><b>{{ .Name }}</b></>
[TRACE] 21:36:22.986 shell.go:TemplateCache() - 177ns
[TRACE] 21:36:22.986 shell.go:Shell() - 52ns
[TRACE] 21:36:22.987 shell.go:TemplateCache() - 669ns
[TRACE] 21:36:22.987 shell.go:Shell() - 252ns
[DEBUG] 21:36:22.987 text.go:Render:70 → Rendering template: plain
[DEBUG] 21:36:22.987 text.go:Render:70 → Rendering template: plain
[DEBUG] 21:36:22.987 text.go:Render:70 → Rendering template: plain
[DEBUG] 21:36:22.987 text.go:Render:70 → Rendering template: plain
[DEBUG] 21:36:22.987 text.go:Render:70 → Rendering template: plain
[TRACE] 21:36:22.987 shell.go:Shell() - 84ns
[TRACE] 21:36:22.987 shell.go:Shell() - 23ns
[TRACE] 21:36:22.987 shell.go:Flags() - 55ns
[TRACE] 21:36:22.987 shell.go:Shell() - 146ns
[TRACE] 21:36:22.987 shell.go:Shell() - 24ns
[DEBUG] 21:36:22.988 shell_unix.go:TerminalWidth:70 → terminal width: 131
[TRACE] 21:36:22.988 shell_unix.go:TerminalWidth() - 792.416µs
[TRACE] 21:36:22.988 shell.go:Flags() - 53ns
[DEBUG] 21:36:22.988 shell.go:Getenv:398 → NO DATA
[TRACE] 21:36:22.988 shell.go:Getenv(OMP_CACHE_DIR) - 3.234µs
[DEBUG] 21:36:22.988 shell.go:Getenv:398 → NO DATA
[TRACE] 21:36:22.988 shell.go:Getenv(XDG_CACHE_HOME) - 1.522µs
[TRACE] 21:36:22.988 shell_unix.go:CachePath() - 16.102µs
[TRACE] 21:36:22.988 shell.go:Flags() - 47ns
```

--- a/website/static/install.sh
+++ b/website/static/install.sh
@@ -81,7 +81,7 @@ validate_install_directory() {
 
     # check if we can write to the install directory
     if [ ! -w $install_dir ]; then
-        error "Cannot write to ${install_dir}. Please run the script with sudo and try again:\n  curl -s https://ohmyposh.dev/install.sh | sudo bash -s\n\nAlternatively, you can set a different directory and try again: \n  curl -s https://ohmyposh.dev/install.sh | bash -s -- -d {directory}"
+        error "Cannot write to ${install_dir}. Please set a different directory and try again: \n  curl -s https://ohmyposh.dev/install.sh | bash -s -- -d {directory}"
     fi
 
     # check if the directory is in the PATH

